client-hc/src/test/java/com/graphhopper/api/GraphHopperWebIT.java

```diff
@@ -67,9 +67,9 @@ public void testSimpleRoute() {
 
     @Test
     public void testAlternativeRoute() {
-        // https://graphhopper.com/maps/?point=52.044124%2C10.378346&point=52.043847%2C10.381994&algorithm=alternative_route&ch.disable=true
+        // https://graphhopper.com/maps/?point=52.044098%2C10.378368&point=52.043847%2C10.381994&algorithm=alternative_route&ch.disable=true
```

Comment:
Here the server did not produce alternative routes anymore, I had to move the waypoint a couple of meters.

Reply:
Hmmh, strange it should still produce alternatives: https://graphhopper.com/maps/?point=52.044124%2C10.378346&point=52.043847%2C10.381994&algorithm=alternative_route&ch.disable=true

Reply:
Mhm, maybe it was a one time issue? Should I revert the change?

Reply:
I checked it on GH Maps and it didn't produce an alternative route.

Reply:
locally it seems to fail - got the same problem in my branch: https://github.com/graphhopper/graphhopper/commits/json_refactoring

Reply:
This is weird as it is okay, when I run it locally.

Reply:
Yes, when rerunning the failing jobs now they become green, i.e. they failed temporary due to some data update.

Reply:
Yes I had the same impression :). Should I revert this change as it doesn't really belong into this PR?

Reply:
ok